pkg/pod_lister/resolve_container.go

```diff
@@ -255,11 +257,17 @@ func getPathFromcGroupID(cgroupId uint64) (string, error) {
 	return cGroupIDToPath[cgroupId], nil
 }
 
+// Get containerID from path. cgroup v1 and cgroup v2 will use different regex
 func extractPodContainerIDfromPath(path string) (string, error) {
+	cgroup := config.GetCGroupVersion()
```

Comment:
to avoid frequent `GetCGroupVersion()` calls, please make a global var for `cgroup` in the `var` block.